src/js/containers/router/RouterRoutes.jsx

```diff
@@ -1,16 +1,17 @@
 import { hashHistory } from 'react-router';
-import LandingPage from '../../components/landing/LandingPage';
-import LoginPage from '../../components/login/LoginPage';
-import AuthPage from '../../components/login/AuthPage';
-import SubmissionGuideContainer from '../../containers/addData/SubmissionGuideContainer';
-import AddDataPageContainer from '../../containers/addData/AddDataPageContainer';
+import LandingPage from 'components/landing/LandingPage';
```

Comment:
This doesn't look like it moved, why do we not need the `../../` anymore?

Reply:
The Webpack updates included resolving `.jsx` and `.js` files to the `src/js` directory, so I'm updating to cleaner imports when I come across them.